certified-connectors/OpenLegacy IBM I (AS400)/apiDefinition.swagger.json

```diff
@@ -1,318 +1,477 @@
+                      "description": "The Contract ID selected by user."
+                  },
+                  {
+                      "name": "enrichmentType",
+                      "in": "query",
+                      "default": "HTTP",
+                      "type": "string",
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines the type of enrichment metadata to be included in the response.",
+                      "description": "Determines the type of enrichment metadata to be included in the response.",
+                      "required": true
+                  },
+                  {
+                      "name": "published",
+                      "type": "boolean",
+                      "in": "query",
+                      "default": true,
+                      "x-ms-visibility": "internal",
+                      "x-ms-summary": "Determines that Published method only will be included in the response.",
+                      "description": "Determines that Published method only will be included in the response.",
+                      "required": true
+                  }
+              ],
+              "x-ms-visibility": "internal",
+              "responses": {
+                  "200": {
+                      "description": "OK",
+                      "schema": {
+                          "$ref": "#/definitions/GetMethodsForContractResponse"
+                      }
+                  }
+              }
+          }
+      },
+      "/api/v1/partners/method-swagger-by-id/{methodId}": {
+          "get": {
+              "summary": "Get Method Open Api Spec",
+              "operationId": "GetMethodOpenApiSpec",
+              "description": "Get a method OpenApi specification",
+              "parameters": [
+                  {
+                      "name": "methodId",
+                      "in": "path",
+                      "required": true,
+                      "type": "string",
+                      "x-ms-url-encoding": "single",
+                      "x-ms-summary": "Method ID for which an OpenApi specification must be returned.",
+                      "description": "Method ID for which an OpenApi specification must be returned."
+                  }
+              ],
+              "x-ms-visibility": "internal",
+              "responses": {
+                  "200": {
+                      "description": "OK",
+                      "schema": {
+                          "type": "object",
+                          "properties": {
+                              "flow-input": {
+                                  "type": "object"
+                              },
+                              "flow-output": {
+                                  "type": "object"
+                              }
+                          }
+                      }
+                  }
+              }
+          }
+      },
+      "/dummy-AS400Cobol": {
```

Comment:
@boiv can you please confirm if the API is in production? we don't allow publishing connectors with API is stagging or testing mode.

Reply:
Yes and no :)
All operations which looks like `/dummy-{operationId}`, are handling by C# script and depends on User choice and User's deployed application URL (provided as part of connector connection).
In the script, we programmatically update a request path, request params and body based on User's choice